UnityProject/Assets/Scripts/Health/Objects/Integrity.cs

```diff
@@ -467,6 +449,46 @@ private void OnDestroy()
 		BeingDestroyed?.Invoke();
 		BeingDestroyed = null;
 	}
+
+	public string GetDamageDesc()
+	{
+		return "Integrity: " + PercentageDamaged switch
+		{
+			< 10 => "Crumbling".Color(Color.red),
+			< 30 => "Heavily Damaged".Color(Color.red),
+			< 40 => "Significantly Damaged".Color(Color.yellow),
+			< 60 => "Worn out Condition".Color(Color.yellow),
+			< 80 => "Slightly Damaged".Color(Color.green),
+			< 95 => "Scratched Condition".Color(Color.green),
+			>= 100 => "In Perfect Condition".Color(Color.green),
```

Comment:
More protips never hurt.
Just some inconsistency in the descriptions however;
You have 'Scratched Condition' & 'In Perfect Condition', I would rename it to 'Perfect Condition' to remain consistent with the other descriptions.
Also in 'Worn out Condition', out isn't capitalized but all other words in these descriptions are.

Reply:
updated